lib/audited/auditor.rb

```diff
@@ -101,9 +101,15 @@ def without_auditing(&block)
       #   end
       #
       def revisions(from_version = 1)
-        audits = self.audits.from_version(from_version)
-        return [] if audits.empty?
-        audits.map(&:revision)
+        return [] if audits.from_version(from_version).empty?
+
+        loaded_audits = audits.select(:audited_changes, :version).to_a
```

Comment:
You can use `audits.pluck(:audited_changes, :version)` here, I think that will work on all rails versions.

Reply:
Yeah, I can, but just don't want to deal with raw arrays and complicate code a bit.